cypress-tests/cypress/support/commands.js

```diff
@@ -3331,3 +3331,95 @@ Cypress.Commands.add("incrementalAuth", (globalState, data) => {
     }
   });
 });
+
+Cypress.Commands.add("createConfigs", (globalState, key, value) => {
+  const base_url = globalState.get("baseUrl");
+  const api_key = globalState.get("adminApiKey");
+
+  cy.request({
+    method: "POST",
+    url: `${base_url}/configs/`,
+    headers: {
+      "Content-Type": "application/json",
+      "api-key": api_key,
+    },
+    body: {
+      key: key,
+      value: value,
+    },
+    failOnStatusCode: false,
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+
+    expect(response.status).to.equal(200);
+    expect(response.body).to.have.property("key").to.equal(key);
+    expect(response.body).to.have.property("value").to.equal(value);
+  });
+});
+
+Cypress.Commands.add("fetchConfigs", (globalState, key, value) => {
+  const base_url = globalState.get("baseUrl");
+  const api_key = globalState.get("adminApiKey");
+
+  cy.request({
+    method: "GET",
+    url: `${base_url}/configs/${key}`,
+    headers: {
+      "Content-Type": "application/json",
+      "api-key": api_key,
+    },
+    failOnStatusCode: false,
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+
+    expect(response.status).to.equal(200);
+    expect(response.body).to.have.property("key").to.equal(key);
+    expect(response.body).to.have.property("value").to.equal(value);
+  });
+});
+
+Cypress.Commands.add("updateConfigs", (globalState, key, value) => {
+  const base_url = globalState.get("baseUrl");
+  const api_key = globalState.get("adminApiKey");
+
+  cy.request({
+    method: "POST",
+    url: `${base_url}/configs/${key}`,
+    headers: {
+      "Content-Type": "application/json",
+      "api-key": api_key,
+    },
+    body: {
+      key: key,
+      value: value,
+    },
+    failOnStatusCode: false,
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+
+    expect(response.status).to.equal(200);
+    expect(response.body).to.have.property("key").to.equal(key);
+    expect(response.body).to.have.property("value").to.equal(value);
+  });
+});
+
+Cypress.Commands.add("deleteConfigs", (globalState, key, value) => {
+  const base_url = globalState.get("baseUrl");
+  const api_key = globalState.get("adminApiKey");
+
+  cy.request({
+    method: "DELETE",
+    url: `${base_url}/configs/${key}`,
+    headers: {
+      "Content-Type": "application/json",
+      "api-key": api_key,
+    },
+    failOnStatusCode: false,
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+
+    expect(response.status).to.equal(200);
+    expect(response.body).to.have.property("key").to.equal(key);
+    expect(response.body).to.have.property("value").to.equal(value);
+  });
+});
```

Comment:
i would suggest you remove this part of code and instead, update the `updateConfig` command at [`L3219`](https://github.com/juspay/hyperswitch/blob/003cf1c1ff60ae642c347fd63d8224da51e082c3/cypress-tests/cypress/support/commands.js#L3219) as previously discussed.

also, make sure you add name to the assertion instead of having it plain. example:

```js
expect(
        response.body.incremental_authorization_allowed,
        "incremental_authorization_allowed" // this is the name to the assertion and having this makes it more meaningful
      ).to.be.true;
```